doc/source/installation.rst

```diff
@@ -47,7 +47,35 @@ In addition
 
 are used to build and test the documentation.
 
-A few other packages such as LaTeX is used for circuit plotting, please refer to the main documentation section for detailed instruction.
+.. _circuit_plot_packages:
+
+Plotting Circuits
+------------------
+In order to plot circuits, following packages are needed - ``LaTex``, ``ImageMagick``,
+``QCircuit``, ``pdfcrop`` and ``pdflatex``.
+
+**For Linux** : If you would prefer to avoid installing the full ``texlive``
+package for ``LaTex``, `this link has <https://tex.stackexchange.com/a/504566/203959>`_
+some useful discussion on selectively installing smaller packages. ``LaTex`` is needed
+to plot the circuits.
+
+* ``braket`` will be installed as a part of ``texlive-latex-extra``
+* ``qcircuit`` will be installed as a part of ``texlive-pictures``
+* ``pdfcrop`` and ``pdflatex`` are installed when a minimal ``texlive`` is installed.
+
+``ImageMagick`` can be installed as a conda package via `this link <https://github.com/conda-forge/imagemagick-feedstock#installing-imagemagick>`_
+or use this `link <https://imagemagick.org/script/download.php>`_ to install
+via source. This package along with ``pdfcrop`` and ``pdflatex`` are
+needed to display the circuit diagrams.
+
+
+You `might need to make changes <https://stackoverflow.com/a/52863413/10241324>`_ to ``policy.xml`` if following error occurs :
+
+.. code-block:: text
+
+  RuntimeError: convert-im6.q16: not authorized `qcirc.pdf' @ error/constitute.c/ReadImage/412.
+  convert-im6.q16: no images defined `qcirc.png' @ error/convert.c/ConvertImageCommand/3258.
+
```

Comment:
I moved the guide of Windows also here. Also added `pdf2svg` as well as tried to emphasize that those are not Python packages. The last point caused quite some trouble for the guy we were helping a few days ago.

How do find the following?

```suggestion
Additional software for Plotting Circuits
=========================================
In order to plot circuits, the following non-Python softwares are needed:

LaTeX
-----
The circuit plotting function in QuTiP uses LaTeX and a few LaTeX packages including ``qcircuit``, ``pdfcrop`` and ``braket``.

**For Linux and Mac** :
You need to install a TeX distribution such as `TeX Live <https://www.tug.org/texlive/>`_. You can either install it through ``apt-get``/``brew`` or using their installer on the website.
If you would prefer to avoid installing the full ``texlive`` distribution (which is a few Gigabyte large), `this link <https://tex.stackexchange.com/a/504566/203959>`_
has some useful discussion on selectively installing part of the packages. In particular:

* ``braket`` will be installed as a part of ``texlive-latex-extra``
* ``qcircuit`` will be installed as a part of ``texlive-pictures``
* ``pdfcrop`` and ``pdflatex`` are installed when a minimal ``texlive`` is installed.

**For Windows** : 
We recommend installing `MiKTeX <https://miktex.org/>`_, which will automatically install necessary packages like `qcircuit` for you when it is used. It will only take a few more minutes in your first attempt at plotting a circuit.
In addition, you also need to install perl for ``pdfcrop``.

ImageMagick and Ghostscript
---------------------------
In order to display the circuit in Jupyter notebook, we need to convert it to png format. To do that, you will need to isntall `Ghostscript <https://www.ghostscript.com/>`_ and `ImageMagick <https://imagemagick.org/script/download.php>`_.
The first is responsible for reading the pdf file while the second converting it to png.

.. note::
    You `might need to make changes <https://stackoverflow.com/a/52863413/10241324>`_ to ``policy.xml`` if the following error occurs :

    .. code-block:: text

        RuntimeError: convert-im6.q16: not authorized `qcirc.pdf' @ error/constitute.c/ReadImage/412.
        convert-im6.q16: no images defined `qcirc.png' @ error/convert.c/ConvertImageCommand/3258.

pdf2svg
-------
To convert the circuit into svg format, you will need to install ``pdf2svg``.
Please visit `their website <https://github.com/dawbarton/pdf2sv>`_ for installation guide.

.. note::
    If you want to check whether all dependencies are installed,
    see if the following three commands work correctly:
    ``pdflatex``, ``pdfcrop`` and ``magick anypdf.pdf anypdf.png``,
    where ``anypdf.pdf`` is any pdf file you have.
```